sysutils/pfSense-pkg-apcupsd/files/usr/local/www/widgets/include/apcupsd_inc.inc

```diff
@@ -0,0 +1,26 @@
+<?php
+/*
+ * apcupsd.widget.php
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2021 Rubicon Communications, LLC (Netgate)
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//set variable for custom title
+$apcupsd_title = gettext("Apcupsd Improved");
```

Comment:
Why the "Improved" here? Seems unnecessary.

Reply:
It is a working title, as in, I improved upon the existing plug-in, I am not married to it.